src/azure-cli/azure/cli/command_modules/acs/_params.py

```diff
@@ -366,6 +369,9 @@ def load_arguments(self, _):
         c.argument('max_count', type=int, validator=validate_nodes_count)
         c.argument('nodepool_labels', nargs='*', validator=validate_nodepool_labels,
                    help='space-separated labels: key[=value] [key[=value] ...]. See https://aka.ms/node-labels for syntax of labels.')
+        c.argument('disable_defender', action='store_true', validator=validate_disable_and_enable_parameters)
+        c.argument('enable_defender', action='store_true')
```

Comment:
We do not recommend adding two completely opposite parameters repeatedly, because it will lead to too many similar parameters of CLI which will increase the learning cost of users. Moreover, it may also lead to conflicting usage of parameters. 
However, as we have discussed with @andyzhangx @ZeroMagic @FumingZhang before, this issue is not blocking us from merging this PR for the time being.  @FumingZhang has a work item to refile these usage places in a unified way in the future. @FumingZhang Please be aware~

Reply:
Yeah, totally agreed. Ack